linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/bash/shared.sh

```diff
@@ -0,0 +1,23 @@
+# platform = multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_sle
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
+
+{{{ bash_instantiate_variables("rsyslog_remote_loghost_address") }}}
+
+# Get omfwd configuration directive
+OMFWD_CONFIG_OUTPUT=`grep -Pzo '(?s)action\s*\(\s*type\s*=\s*"omfwd".*\)' /etc/rsyslog.conf /etc/rsyslog.d/*.conf`
+OMFWD_CONFIG=`echo "$OMFWD_CONFIG_OUTPUT"| awk 'BEGIN {FS=":"; RS=")\n"}; {print $2}'`
+OMFWD_CONFIG_FILE=`echo OMFWD_CONFIG_OUTPUT| awk 'BEGIN {FS=":"; RS=")\n"}; {print $1}'`
```

Comment:
probably a missing $ before OFMD_CONFIG_OUTPUT

Reply:
:bow: should be ok in 79dfba1825e